src/components/Films/Films.tsx

```diff
@@ -44,6 +43,26 @@ const Films = () => {
     getFilmInfo();
   }, [title]);
 
+  const columns = [
+    { title: 'Original Title', dataIndex: 'original_title', key: 'original_title' },
+    { title: 'Original Language', dataIndex: 'original_language', key: 'original_language' },
+    { title: 'Release Date', dataIndex: 'release_date', key: 'release_date' },
+    { title: 'Vote Average', dataIndex: 'vote_average', key: 'vote_average' },
+    { title: 'Overview', dataIndex: 'overview', key: 'overview' },
+    {
+      title: 'Poster',
+      dataIndex: 'poster_path',
+      key: 'poster_path',
+      render: (Poster: any) => {
+        return <img src={`https://image.tmdb.org/t/p/w500/${Poster}`} style={{ width: '250px' }} />;
```

Comment:
do not use inline style and btw I do not think that hardcoded px is good maybe some percent or whatever ?